shub/image/utils.py

```diff
@@ -31,10 +31,13 @@
 """
 
 
-def debug_log(msg):
+def is_verbose():
     ctx = click.get_current_context(True)
-    # FIXME remove check of debug param as it's deprecated
-    if ctx and (ctx.params.get('verbose') or ctx.params.get('debug')):
+    return ctx and ctx.params.get('verbose')
```

Comment:
I removed it here because I had a feeling that using `debug` parameter is already disabled and doesn't work except for raising a warning: this [function](https://github.com/scrapinghub/shub/blob/68623011256b396954b278656f5a0ac919f3cb24/shub/image/utils.py#L40) doesn't return anything, so context doesn't contain `debug` parameter. If this is considered as a wrong behaviour, let's fix callback logic.